lib/course_planner_web/templates/event/form.html.eex

```diff
@@ -0,0 +1,32 @@
+  <div class="row">
+    <div class="col-xs-12 col-md-6 col-md-offset-3">
+      <%= form_for @changeset, @action, fn f -> %>
+        <%= CoursePlannerWeb.SharedView.card @title do %>
+          <%= CoursePlannerWeb.SharedView.card_content do %>
+            <%= if @changeset.action do %>
+              <div class="form-error">
+                <p>Oops, something went wrong! Please check the errors below.</p>
+              </div>
+            <% end %>
+            <%= CoursePlannerWeb.SharedView.form_text f, :name, required: true %>
+            <%= CoursePlannerWeb.SharedView.form_text f, :location, required: true %>
+            <%= CoursePlannerWeb.SharedView.form_date f, :date, required: true %>
+            <%= CoursePlannerWeb.SharedView.form_time f, :starting_time, required: true, value: %{hour: 8, min: 0}  %>
+            <%= CoursePlannerWeb.SharedView.form_time f, :finishing_time, required: true, value: %{hour: 8, min: 0}  %>
+            <%= CoursePlannerWeb.SharedView.form_textarea f, :description %>
+            <%= CoursePlannerWeb.SharedView.form_multiselect f, :user_ids, all_users(), selected: selected_users(@changeset),
```

Comment:
this really needs a Add_all functionality. I know this does not fall in this PR. but seriously creating an event for many users is gonna be a pain in the ass

Reply:
We definitely do.